packages/fx-core/templates/plugins/resource/sql/bicep/sql.template.bicep

```diff
@@ -0,0 +1,48 @@
+param sqlServerName string
+param sqlDatabaseName string 
+param administratorLogin string
+param administratorLoginPassword string
+param AADUser string
+param AADObjectId string
+param AADTenantId string
+
+resource sqlServer 'Microsoft.Sql/servers@2021-02-01-preview' = {
+  location: resourceGroup().location
+  name: sqlServerName
+  properties: {
+    administratorLogin: administratorLogin
+    administratorLoginPassword: administratorLoginPassword
+  }
+}
+
+resource sqlServerAAD 'Microsoft.Sql/servers/administrators@2021-02-01-preview' = {
```

Comment:
Since this is only used in postProvision to grant permissions, should we set AAD admin here? We might meet errors if users changed the admin settings here.

Reply:
Sure